packages/plugin-compiler/src/main/kotlin/io/realm/compiler/Identifiers.kt

```diff
@@ -78,7 +80,8 @@ internal object FqNames {
     val REALM_REFERENCE = FqName("io.realm.internal.RealmReference")
     val REALM_MEDIATOR_INTERFACE = FqName("io.realm.internal.Mediator")
     val REALM_CLASS_IMPL = FqName("io.realm.internal.schema.RealmClassImpl")
-    val REALM_CONFIGURATION = FqName("io.realm.RealmConfiguration")
+    val CLASS_METADATA_CLASS = FqName("io.realm.internal.schema.ClassMetadata")
+    val REALM_LOCAL_CONFIGURATION = FqName("io.realm.LocalConfiguration")
```

Comment:
Unused, same for `SYNC_CONFIGURATION`, `REALM_CONFIGURATION_BUILDER`, `SYNC_CONFIGURATION_BUILDER`  and other FqNames ...